pull.go

```diff
@@ -59,18 +59,46 @@ type PullParams struct {
 	LocaleID string
 }
 
-func (params *PullParams) UnmarshalYAML(unmarshal func(interface{}) error) error {
-	var aux struct {
-		LocaleID string `yaml:"locale_id,omitempty"`
-	}
-	if err := unmarshal(&aux); err != nil {
+func (tgt *Target) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	m := map[string]interface{}{}
+	if err := unmarshal(m); err != nil {
 		return err
 	}
-	params.LocaleID = aux.LocaleID
 
-	return unmarshal(&params.LocaleDownloadParams)
-}
+	var ok bool
+	cfgErrStr := "configuration key %q has invalid value\nsee https://phraseapp.com/docs/developers/cli/configuration/"
+	for k, v := range m {
+		switch k {
+		case "file":
+			if tgt.File, ok = v.(string); !ok { return fmt.Errorf(cfgErrStr, k) }
```

Comment:
I would use something like `validateIsString()` which then returns the `cfgErrStr` (which then needs to be moved to e.g. a global `var`)